tools/clang/test/CodeGenDXIL/hlsl/types/matrix/default-matrix-in-template.hlsl

```diff
@@ -9,7 +9,6 @@ StructuredBuffer<matrix> buf1;
 
 RWBuffer<float4> buf2;
 
-[RootSignature("DescriptorTable(SRV(t0), UAV(u0))")]
```

Comment:
Just to document this. This is a workaround for #5476. Since the root signature isn't really part of this test, we decided it was okay to remove this line to keep the test running